mandatory/4-tax.js

```diff
@@ -5,7 +5,11 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+    const taxChargeAmount = price + (20 / 100) * price
```

Comment:
Sometimes you have to avoid declaring too much variables in order to shorten your code. Try putting the "method operation" statement on the return statement if its short enough

Reply:
I'll shorten it right away.

Reply:
Done, thanks for the feedback.